index.html

```diff
@@ -11,174 +11,183 @@
+          </tr>
+          <tr>
+            <td data-label="Navn">Christian Andersen</td>
+            <td data-label="Company">Pixel Inc.</td>
+            <td data-label="Level">Senior</td>
+            <td data-label="Deals Closed">315</td>
+            <td data-label="State">California</td>
+            <td data-label="Phone">+1764156161</td>
+            <td data-label="Email">chandersen@pixel.com</td>
+          </tr>
+          <tr>
+            <td data-label="Navn">Christian Andersen</td>
+            <td data-label="Company">Pixel Inc.</td>
+            <td data-label="Level">Senior</td>
+            <td data-label="Deals Closed">315</td>
+            <td data-label="State">California</td>
+            <td data-label="Phone">+1764156161</td>
+            <td data-label="Email">chandersen@pixel.com</td>
+          </tr>
+          <tr>
+            <td data-label="Navn">Christian Andersen</td>
+            <td data-label="Company">Pixel Inc.</td>
+            <td data-label="Level">Senior</td>
+            <td data-label="Deals Closed">315</td>
+            <td data-label="State">California</td>
+            <td data-label="Phone">+1764156161</td>
+            <td data-label="Email">chandersen@pixel.com</td>
+          </tr>
+          <tr>
+            <td data-label="Navn">Christian Andersen</td>
+            <td data-label="Company">Pixel Inc.</td>
+            <td data-label="Level">Senior</td>
+            <td data-label="Deals Closed">315</td>
+            <td data-label="State">California</td>
+            <td data-label="Phone">+1764156161</td>
+            <td data-label="Email">chandersen@pixel.com</td>
+          </tr>
+          <tr>
+            <td data-label="Navn">Christian Andersen</td>
+            <td data-label="Company">Pixel Inc.</td>
+            <td data-label="Level">Senior</td>
+            <td data-label="Deals Closed">315</td>
+            <td data-label="State">California</td>
+            <td data-label="Phone">+1764156161</td>
+            <td data-label="Email">chandersen@pixel.com</td>
+          </tr>
+          <tr>
+            <td>
+              Total agents: 28
+            </td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td colspan="3">
+              <div class="table-footer">
+                Resultater per side:
+                <select>
+                  <option value="6">6</option>
+                </select>
+                <span>
+                          Side 1 av 5
+                        </span>
+                <div class="chevron-buttons">
+                  <div class="chevron"><</div>
+                  <div class="chevron chevron-active">></div>
+                </div>
               </div>
-            </div>
-          </td>
-        </tr>
-        </tbody>
-      </table>
-    </div>
-  </article>
+            </td>
+          </tr>
+          </tbody>
+        </table>
+      </div>
+    </article>
+  </div>
 </section>
 
 <footer>
```

Comment:
Footer should also have horizontal gutters. Since there is already container "tm-container" in header and content additional in footer would be a reasonable choice.